content/kubermatic/master/tutorials_howtos/networking/ipam/_index.en.md

```diff
@@ -0,0 +1,157 @@
+- IPv4 `poolCIDR` should have a prefix (i.e. mask) equal or greater than **20**.
+- IPv6 `poolCIDR` should have a prefix (i.e. mask) equal or greater than **116**.
+
+For the "prefix" allocation type:
+- `allocationPrefix` should be between **1** and **32** for IPv4 pool, and between **1** and **128** for IPv6 pool.
+- `allocationPrefix` should be equal or greater than the pool subnet mask size.
+
+### Modifications
+In general, modifications are not allowed. If you need to change an already applied `IPAMPool`, you should first delete it (note that all user clusters allocations `IPAMAllocation` will be deleted, in that case) and then apply it with the changes.
+
+The only allowed modification in a `IPAMPool` CR is the deletion of a datacenter configuration if there is no persisted allocation `IPAMAllocation` in any user cluster for it.
+
+## Generated resource (IPAMAllocation)
+The IPAM controller in the seed-controller-manager is in charge of the allocation of IP ranges from the defined pools for user clusters. For each user cluster which runs in a datacenter for which an `IPAMPool` is defined, it will automatically allocate a free IP range from the available pool.
+
+The persisted allocation is an `IPAMAllocation` CR that will be installed in the seed cluster in the user cluster's namespace.
+
+E.g. for "prefix" type:
+```yaml
+apiVersion: kubermatic.k8c.io/v1
+kind: IPAMAllocation
+metadata:
+  name: metallb
+  namespace: cluster-kd8jnt7gjj
+spec:
+  cidr: "192.168.1.0/30"
+  dc: aws-eu-central-1a
+  type: prefix
+```
+
+E.g. for "range" type:
+```yaml
+apiVersion: kubermatic.k8c.io/v1
+kind: IPAMAllocation
+metadata:
+  name: metallb
+  namespace: cluster-sd87xtqpnm
+spec:
+  addresses:
+  - "192.168.1.0-192.168.1.7"
+  dc: azure-westeurope
+  type: range
+```
+
+Note that the ranges of addresses may be disjoint for the "range" type, e.g.:
+```yaml
+spec:
+  addresses:
+  - "192.168.1.0-192.168.1.7"
+  - "192.168.1.16-192.168.1.23"
+```
+That's because, in the future, we could start allowing the modification (i.e. increase) of the allocation range.
+
+### Allocations cleanup
+The allocations (i.e. `IPAMAllocation` CRs) for a user cluster are deleted in two occasions:
+- Related pool (i.e. `IPAMPool` CR with same name) is deleted.
+- User cluster itself is deleted.
+
+## KKP Addon template integration
+The user cluster allocated ranges (i.e. `IPAMAllocation` CRs values) are available in the Addon template data (attribute `.Cluster.Network.IPAMAllocations`) to be rendered in the Addons manifests.
+
+E.g. looping all user cluster IPAM pools allocations:
+```yaml
+...
+
+{{- range $ipamPool, $allocation := .Cluster.Network.IPAMAllocations }}
+{{ $ipamPool }}:
+  {{- if eq $allocation.Type "prefix" }}
+  CIDR: {{ $allocation.CIDR }}
+  {{- end }}
+  {{- if eq $allocation.Type "range" }}
+  Addresses:
+    {{- range $allocation.Addresses }}
+    - {{ . }}
+    {{- end }}
+  {{- end }}
+{{- end }}
+```
+
+## MetalLB Addon
```

Comment:
Could you please extend this with regard to the latest addon change - e.g. requirement for and IPAMPool with a specific name?